spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -158,6 +158,428 @@ Make sure to add 'spring-boot-configuration-processor' as an optional dependency
 ----
 ====
 
+[[custom-applications]]
```

Comment:
This can be simplified by pointing to the stream documentation for details on how to actually create Spring Cloud Stream applications. Also, registration of custom apps and creating streams is already covered by other sections in the documentation, so not sure why do we need this content again ? A simple paragraph pointing to the SCSt app tutorial should suffice ?